release-tools/scripts/prod-sync-artifacts.py

```diff
@@ -0,0 +1,265 @@
+                            plugin_fullpath = rc_plugin_path + plugin_category + "/"
+                            if plugin_build is not None:
+                                plugin_val = check_plugin(rc_bucket,plugin_fullpath,
+                                    plugin_name,plugin_version,plugin_build,spec)
+                                if plugin_val:
+                                    print("Plugin : " + plugin_val.split('/')[-1] + "\t Status : Found")
+                                    plg_state = "Found"
+                                    plg_name = plugin_val.split('/')[-1]
+                                    final_name = re.sub('-build-[0-9]*','',plg_name)
+                                    print("Plugin final name :  " + final_name)
+                                    if action == "prod-sync":
+                                        upload_artifact(rc_bucket,plugin_val,plg_prd_bucket,plg_prd_path)
+                                else:
+                                    plg_name = "NA"
+                                    final_name = plugin_name
+                                    plg_state = "Not Found"
+                                    print("Plugin : " + plugin_name + "-" + plugin_version + "\t Status : Not Found")
+                            else:
+                                print("Build number is missing for plugin : " + plugin_name)
+                                raise
+                            artifact_status.append([final_name,plg_name,plg_state])
+                except Exception as ex:
+                    print(ex)
+            print("\n\n\n\n")
+            odfe_es = [["x64.tar.gz","tarball/opendistro-elasticsearch/"],
+                       ["arm64.tar.gz","tarball/opendistro-elasticsearch/"],
+                       ["exe","downloads/odfe-windows/odfe-executables/"],
+                       ["zip","downloads/odfe-windows/ode-windows-zip/"],
+                       ["x64.rpm","downloads/rpms/opendistroforelasticsearch/"],
+                       ["arm64.rpm","downloads/rpms/opendistroforelasticsearch/"],
+                       ["x64.deb","downloads/debs/opendistroforelasticsearch/"],
+                       ["arm64.deb","downloads/debs/opendistroforelasticsearch/"]]
+            odfe_kb = [["x64.tar.gz","tarball/opendistroforelasticsearch-kibana/"],
+                       ["arm64.tar.gz","tarball/opendistroforelasticsearch-kibana/"],
+                       ["exe","downloads/odfe-windows/odfe-executables/"],
+                       ["zip","downloads/odfe-windows/ode-windows-zip/"],
+                       ["x64.rpm","downloads/rpms/opendistroforelasticsearch-kibana/"],
+                       ["arm64.rpm","downloads/rpms/opendistroforelasticsearch-kibana/"],
+                       ["x64.deb","downloads/rpms/opendistroforelasticsearch-kibana/"],
+                       ["arm64.deb","downloads/rpms/opendistroforelasticsearch-kibana/"]]
```

Comment:
Why hardcoding instead of using the manufest.yml?

Reply:
As we are in the middle of the release so, didnt want to introduce new changes to Manifest.